src/main/java/de/zalando/aruha/nakadi/enrichment/EnrichmentStrategy.java

```diff
@@ -0,0 +1,9 @@
+package de.zalando.aruha.nakadi.enrichment;
+
+import de.zalando.aruha.nakadi.domain.EventType;
+import de.zalando.aruha.nakadi.exceptions.EnrichmentException;
+import org.json.JSONObject;
+
+public interface EnrichmentStrategy {
+    JSONObject enrich(JSONObject event, EventType eventType) throws EnrichmentException;
```

Comment:
Here as well, as the original event will be enriched I don't see any value to also return it. When I look at such method it makes me think that original event will not be modified (as it has a return value).